### PR TITLE
CUDOS-434: Implementation of error handling

### PR DESCRIPTION
**Pull request by aemil145**
In addition to error handling, these changes contain minor refactoring for better code readability

### CODE REVIEW

**Review by Holymir — APPROVED** [review]

**Review by tgntr — APPROVED** [review]

File: packages/cudos-cmd/node/node.js [hunk 1]
```diff
@@ -10,25 +15,19 @@ const startNodeCmd = async function(argv) {
...
 }
 
 const statusNodeCmd = async function() {
   const nodeStatus = await getStatusNode()
   if (nodeStatus.isConnected) {
-    console.log('Connection to node is online.')
-    console.log('Node id: ' + nodeStatus.nodeInfo.nodeId + '\nNetwork: ' + nodeStatus.nodeInfo.network)
-    return
+    console.log('Node is online.')
+  } else {
```
Comment (maptuhec) [inline packages/cudos-cmd/node/node.js]: `return` without `else`

File: packages/cudos-utilities/get-node-status.js [hunk 2]
```diff
@@ -28,35 +28,33 @@ async function getStatusNodeByUrl(url) {
...
     // status is not 200
     nodeStatus.isConnected = false
+    nodeStatus.message = 'Unusual node response. Status code should be 200.'
     return nodeStatus
   } catch (ex) {
-    nodeStatus.isConnected = false
-    if (typeof ex.code !== 'undefined') {
-      nodeStatus.statusCode = ex.code
-      return nodeStatus
+    if (!ex.isAxiosError) {
```
Comment (maptuhec) [inline packages/cudos-utilities/get-node-status.js]: didn't we come to an agreement that we would. use `error` isntead of exception

**Review by maptuhec — APPROVED** [review]

File: packages/cudos-cmd/node/node.js (hunk 1 above)
Comment (aemil145) [reply packages/cudos-cmd/node/node.js]: Now we need to check status message on both node status online and offline. If done with `return` , `if (typeof nodeStatus.message !== 'undefined')` have to be repeated

File: packages/cudos-utilities/get-node-status.js (hunk 2 above)
Comment (aemil145) [reply packages/cudos-utilities/get-node-status.js]: I missed that, fixing it right away

File: packages/cudos-cmd/node/node.js (hunk 1 above)
Comment (maptuhec) [reply packages/cudos-cmd/node/node.js]: I would stick  with 2 `if` statements in this case
`if (node is connected) {
do smth}
if (node is not connected) {
do smth}
`

**Review by maptuhec — CHANGES_REQUESTED** [review]

**Review by maptuhec — APPROVED** [review]

**Pull request merged**